### DISCUSSION TITLE
[Feature]: Light mode for those of us who cannot see in the dark

### DISCUSSION THREAD

**Discussion by wajeht** [coollabsio/coolify#1751] [Feature Requests]
I don't know if there is a setting for it. I cannot find it in the Docs or anywhere.

**Comment by AlejandroAkbal** [comment]
As a workaround you can use an extension like dark reader to make it light themed

**Comment by robsontenorio** [comment]
Please!

**Comment by andrasbacsai** [comment]
It is coming with the "great redesign" that I am working on right now.

**Reply by wajeht** [reply]
hell yeah brother!

**Comment by robsontenorio** [comment]
I am a newcomer. But the v3 interface I see on YouTube videos (months ago) seems more intuitive than this on v4 beta that I have tried.

**Comment by andrasbacsai** [comment]
Now you can wear sunglasses since v244.

**Reply by wajeht** [reply]
thanks bud! 👍
